paradox-menu.el

```diff
@@ -223,26 +223,24 @@ Return (PKG-DESC [STAR NAME VERSION STATUS DOC])."
         (push (cons :stars counts) (package-desc-extras pkg-desc))))
     (list pkg-desc
           `[,(concat
-              (propertize name
-                          'font-lock-face 'paradox-name-face
+              (truncate-string-to-width
+               (propertize name
+                           'font-lock-face 'paradox-name-face
+                           'button t
+                           'follow-link t
+                           'help-echo (format "Package: %s" name)
+                           'package-desc pkg-desc
+                           'action 'package-menu-describe-package)
+               (- paradox-column-width-package button-length) 0 nil t)
+              (make-string (max 0 (- paradox-column-width-package name-length button-length)) ?\s)
```

Comment:
Also, in the `let` above, change `(button-length (length paradox-homepage-button-string))` to `(button-length (if paradox-use-homepage-buttons 0 (length paradox-homepage-button-string)))`.